pyproject.toml

```diff
@@ -15,6 +15,13 @@ packages = [
     { include = "tests", format = "sdist" }
 ]
 
+[tool.poetry.urls]
+"Mailing lists" = "https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information"
+"Code of Conduct" = "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
```

Comment:
I wonder if we could add some tracking here...

Reply:
I would avoid tracking if it is not strictly necessary.

Reply:
Well. Maybe it's not necessary but is useful. Let me expand that by tracking I don't mean tracking individuals but mostly something to help marketing and the docs people figure out where the users come from.

For example, here's what I've been doing in the past: https://github.com/ansible-collections/community.general/blob/ee23c26/.github/ISSUE_TEMPLATE/documentation_report.yml#L107 / https://github.com/ansible/ansible/blob/38a11c3/.github/ISSUE_TEMPLATE/config.yml#L5.

I think these are pretty harmless and don't contribute to the loss of privacy of individuals.

cc @gundalow do you have any thoughts on this?

Reply:
If Felix is happy, we can add it, otherwise I'm not going to push for it.

Reply:
Okay, @felixfontein it's your call then. Feel free to merge.

Reply:
I usually invent the values by example + context